ethers-ext/example/browser-react/src/Util.tsx

```diff
@@ -0,0 +1,24 @@
+import { Account } from "./types";
+
+export function isKaikas(account: Account) {
+    // @ts-ignore
+    return account.provider && account.provider.provider.isKaikas;
+  }
+
+export async function doSendTx(account: Account, makeTxRequest: any) {
+    try {
+        const provider = account.provider;
```

Comment:
The compiler warning says "account.provider is possibly undefined."
Just add a check before using account.provider.

```ts
// here, `account.provider` type is `Web3Provider | undefined`
if (!account.provider) {
  // The compiler learns that if account.provider === undefined, the code stops here.
  throw new Error("wallet not connected")
}
// After this branch, the compiler knows for sure that `account.provider` type is `Web3Provider`
```

Reply:
I updated, thank you.